src/netstandard/ref/System.Security.Cryptography.cs

```diff
@@ -10,6 +10,18 @@ public abstract partial class Aes : System.Security.Cryptography.SymmetricAlgori
         public static new System.Security.Cryptography.Aes Create() { throw null; }
         public static new System.Security.Cryptography.Aes Create(string algorithmName) { throw null; }
     }
+    public sealed partial class AesCcm : System.IDisposable
```

Comment:
These two classes have effectively the same members, but don't implement any kind of common interface or have a common base class. Is that definitely okay?

Reply:
Law of small numbers :smile:.  AES-GCM-SIV as an algorithm merges the tag and ciphertext and doesn't take a nonce... but is also an Authenticated Encryption with Associated Data (AEAD) algorithm (and an AES-AEAD algorithm).

One could handwave and say that when calling Encrypt one MUST pass `ReadOnlySpan<byte>.Empty` for the nonce and `Span<byte>.Empty` for the tag; but that feels a bit too Win32 to me.

(And we could always add an interface or hierarchy later)

Reply:
> (And we could always add an interface or hierarchy later)

That's key part to me. Also, there is the rule of only adding abstractions when there are scenarios for them.